db/schema.rb

```diff
@@ -126,8 +126,8 @@
     t.bigint "user_id"
     t.text "note"
     t.boolean "status", default: true, null: false
-    t.datetime "created_at", precision: 6, null: false
-    t.datetime "updated_at", precision: 6, null: false
+    t.datetime "created_at", null: false
```

Comment:
Any idea what's the reason for those unrelated changes?

Reply:
Seems related to this https://stackoverflow.com/questions/71411355/rails-7-0-schema-has-arbitrary-precision-6-added-on-my-timestamps

Reply:
Trying to address this at https://github.com/rubygems/rubygems.org/pull/3394.